src/content/en/updates/2019/06/image-support-for-async-clipboard.md

```diff
@@ -0,0 +1,299 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Async Clipboard API, the text-focused portion of which we’ve shipped in Chrome 66, has learned a new trick. As of Chrome 76, it now also handles images.
+
+{# wf_published_on: 2019-06-30 #}
+{# wf_updated_on: 2019-06-30 #}
+{# wf_featured_image: /web/updates/images/generic/photo.png #}
+{# wf_tags: chrome76,cutandcopy,execcommand,input,clipboard #}
+{# wf_featured_snippet: The Async Clipboard API, the text-focused portion of which we’ve shipped in Chrome 66, has learned a new trick. As of Chrome 76, it now also handles images. #}
+{# wf_blink_components: Platform>Apps>AppLauncher>Install #}
+
+# Image Support for the Async Clipboard API {: .page-title }
+
+{% include "web/_shared/contributors/thomassteiner.html" %}
+
+In Chrome&nbsp;66, we shipped the [text-focused portion](/web/updates/2018/03/clipboardapi)
+of the Asynchronous [Clipboard API](https://w3c.github.io/clipboard-apis/)
+that replaced the previous method that was based on
+[`Document.execCommand()`](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand)
+to [cut and copy](/web/updates/2015/04/cut-and-copy-commands) contents on the web.
+Now in Chrome&nbsp;76, we are shipping the *image-focused* portion of the Asynchronous Clipboard
+API.
```

Comment:
Potentially obvious addition recommendation:
Could we maybe add a sentence just stating that "this will allow sites to programmatically copy and paste image/png images"? 

While it may seem a bit obvious, and is covered in the linked sites, this was essentially what devs were looking for (also, it might be nice for them to find out early that for now only image/png is supported, as some outspoken devs have asked for image/svg, and assume that "images" includes all/most image formats).

Reply:
This recommended text is very similar to the text of the linked bug, so that people may be reminded of it when reading the article. While adding just one type shouldn't normally be so significant to be pointed out in the 1st paragraph, the bug had a lot of stars so I presume some people may just want to see that this was fixed :P

(Also, thank you for pointing out that more types are being added + raw clipboard access effort in the bottom, for those with more follow-up questions)

Reply:
I have clarified the `image/png` limitation.